torch/_decomp/decompositions.py

```diff
@@ -1361,6 +1361,56 @@ def addmv(self: Tensor, mat1: Tensor, vec: Tensor, beta: int = 1, alpha: int = 1
     return out + beta * self
 
 
+@register_decomposition(aten.pixel_shuffle)
+@out_wrapper()
+@pw_cast_for_opmath
+def pixel_shuffle(self: Tensor, upscale_factor: int):
```

Comment:
Can you move these to `torch/_refs/nn/functional/__init__.py` and add `PythonRefInfo`s so they're fully tested including strides.